### PR TITLE
Migrate Hungary to holiday groups, add l10n support

### PR DESCRIPTION
**Pull request by KJhellico**
## Proposed change

Migrate Hungary to `holiday_groups` standard, localization (en_US, hu, uk).

## Type of change

- [ ] New country/market holidays support (thank you!)
- [ ] Supported country/market holidays update (calendar discrepancy fix, localization)
- [x] Existing code/documentation/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency upgrade (version update)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new python-holidays functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [x] This PR is filed against `beta` branch of the repository
- [x] This PR doesn't contain any merge conflicts and has clean commit history
- [x] The code style looks good: `make pre-commit`
- [x] All tests pass locally: `make test`, `make tox` (we strongly encourage adding tests to your code)
- [x] The related [documentation][docs] has been added/updated (check off the box for free if no updates is required)

[contributing-guidelines]: https://github.com/dr-prodigy/python-holidays/blob/beta/CONTRIBUTING.rst
[docs]: https://github.com/dr-prodigy/python-holidays/tree/beta/docs/source

### CODE REVIEW

File: holidays/locale/uk/LC_MESSAGES/HU.po [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<12 || n%100>14) ? 1 : 2);\n"
+"Generated-By: pygettext.py 1.5\n"
+"X-Generator: Poedit 3.2.2\n"
+
+#. Day off before
+#: ./holidays/countries/hungary.py:45 ./holidays/countries/hungary.py:72
+#, c-format
+msgid "%s előtti pihenőnap"
+msgstr "%s (вихідний)"
```
Comment (KJhellico) [inline holidays/locale/uk/LC_MESSAGES/HU.po:23]: I was unable to find good equivalents for "Day off before/after", considering the problem of cases. So I left one option for both.

Comment (arkid15r) [inline holidays/locale/uk/LC_MESSAGES/HU.po:23]:
```suggestion
msgstr "%s (вихідний за день до)"
```

Comment (arkid15r) [reply holidays/locale/uk/LC_MESSAGES/HU.po:23]: This option is also okay 👍

File: holidays/locale/uk/LC_MESSAGES/HU.po [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+#. Day off after
+#: ./holidays/countries/hungary.py:51
+#, c-format
+msgid "%s utáni pihenőnap"
+msgstr "%s (вихідний)"
```
Comment (arkid15r) [inline holidays/locale/uk/LC_MESSAGES/HU.po:29]:
```suggestion
msgstr "%s (вихідний після)"
```

**Review by arkid15r — COMMENTED** [review]
LGTM

**Pull request merged**